config/default.yml

```diff
@@ -721,6 +721,16 @@ RSpec/FactoryBot/AttributeDefinedStatically:
   VersionChanged: '2.0'
   StyleGuide: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/FactoryBot/AttributeDefinedStatically
 
+RSpec/FactoryBot/ConsistentParenthesesStyle:
+  Description: Use a consistent style for parentheses in factory bot calls.
```

Comment:
I suggest limiting this cop to [directories with factory definitions](https://github.com/rubocop-hq/rubocop-rspec/blob/7d048b653b73315509600ae42241bdb03a033ae4/config/default.yml#L716), for other cases, including specs, there's a RuboCop core cop that keeps an eye on parenthesis usages.

Or do you think that another cop is usually disabled for directories with specs?